src/Application/Redux/reducers.js

```diff
@@ -0,0 +1,80 @@
+import { CLICK_MONEY, COLLECT_INCOME, BUY_ASSET, UPGRADE_CURRENCY, BROADCAST_NEWS } from './actions';
+import StateUtils from './StateUtils';
+
+function trumpMM(state = StateUtils.getInitialState(), action) {
+  switch (action.type) {
+
+    case CLICK_MONEY:
+      return Object.assign({},
+        state,
+        {
+          money: StateUtils.createMoneyAfterClick(state),
```

Comment:
yeah another bump for something like `AddClickMoney`, just a little change but i am prepared to die on this hill